src/components/footer.jsx

```diff
@@ -7,6 +8,20 @@ export function Footer({
   onLogIn,
   onLogOut,
 }) {
+  function AuthButton() {
+    if (loggedIn) {
+      return (
+        <button type="button" onClick={onLogOut} className="logInOutBtn btn">
+          {`${user.name} Logout`}
```

Comment:
Given the `user` prop defined for the `Footer` component, can we just try the following?
```suggestion
          {user.name} Logout
```

Reply:
Implemented.